front/src/applications/operationalStudies/hooks/useSimulationResults.ts

```diff
@@ -124,9 +123,9 @@ const useSimulationResults = (infraId: number): SimulationResults | undefined =>
       .add(selectedOccurrenceIndex * pacedTrainIntervalInMs, 'ms')
       .toISOString();
 
-    const updatedSelectedPacedTrain: PacedTrainWithPacedTrainId = {
+    const updatedSelectedPacedTrain: Train = {
```

Comment:
We could also rename for the selected item, path and simulation ?
```suggestion
    const updatedSelectedOccurrence: Train = {
```

Reply:
It will be modified in the next PR :)